go/vt/wrangler/materializer_test.go

```diff
@@ -1480,11 +1600,34 @@ func TestCreateLookupVindexFailures(t *testing.T) {
 		},
 		err: `vindexType "lookup_noexist" not found`,
 	}, {
-		description: "only one table",
+		description: "only one or two table",
```

Comment:
I'd also prefer to add clarity here with ~ `only source and optionally target tables` and similar in the error message below.